src/main/java/brave/opentracing/BraveTracer.java

```diff
@@ -52,7 +54,7 @@
  * @see BraveSpan
  * @see Propagation
  */
-public final class BraveTracer implements Tracer {
+public final class BraveTracer extends ThreadLocalActiveSpanSource implements Tracer {
```

Comment:
Followup question: is it reasonable/acceptable to expect that spans in a given application are all managed (activated and deactivated) either entirely using Brave or entirely using OT?

I've only got an hour into this so far, but it seems to me that OT's baked-in concept of a "stack" of spans, wherein the deactivation of one leads to the reactivation of whatever was active before it, will require that all activation/deactivation be done via the OT API.